doc/tutorials/space_shooter/step_1.md

```diff
@@ -70,21 +65,26 @@ class SpaceShooterGame extends FlameGame {
     );
   }
 }
-```''',
-  '''
+```
+
 If you run this, you will now see a white rectangle being rendered in the center of the screen.
 
 A couple of points worth commenting:
 
- - Always call `await super.onLoad()` as the first instruction of your custom `onLoad` method.
- - `size` is a `Vector2` variable from the game class and it holds the current dimension of the game
-area, where `x` is the horizontal dimension, or the width, and `y` the vertical dimension, or the height.
- - By default, Flame follows Flutter's canvas anchoring, which means that (0, 0) is anchored on the
+- Always call `await super.onLoad()` as the first instruction of your custom `onLoad` method.
```

Comment:
This is no longer true; the default implementation of `onLoad` does nothing.
If some component does something non-trivial in its `onLoad` method, then it would mark itself as `@mustCallSuper`, and the user will get a warning from the IDE if they don't call `super.onLoad()`.